lib/cog_cmd/ecs/task/list.rb

```diff
@@ -43,6 +44,8 @@ def fetch_task_definitions(client, params, next_token = nil, defs = [])
     end
 
     def process_definition_arns(arns)
+      return unless arns.compact.length > 0
```

Comment:
If there are no `arns` we get an array with one value `nil` set. It only came up during testing, I'm just creating a new instance of the result object manually, but I didn't think it would hurt to put a `compact` in here.